### PR TITLE
Category name letter casing change not allowed when renaming

### PR DESCRIPTION
**Pull request by d-najd**
For more info #3465.

### NOTE

The actual behaviour altered is in `RenameCategory` but because of the renaming it is not included in the commit.

### CODE REVIEW

File: app/src/main/java/eu/kanade/domain/category/model/Category.kt [hunk 1]
```diff
@@ -18,5 +18,9 @@ data class Category(
 }
 
 internal fun List<Category>.anyWithName(name: String): Boolean {
```
Comment (arkon) [inline app/src/main/java/eu/kanade/domain/category/model/Category.kt:20]: Could just add an argument (`ignoreCase = false`)

Comment (d-najd) [reply app/src/main/java/eu/kanade/domain/category/model/Category.kt:20]: Sure the change are up!

**Comment by stevenyomi** [comment]
> including gradle.properites.swp

Why?

**Comment by d-najd** [comment]
> > including gradle.properites.swp
> 
> Why?

if you don't include it the PR build check would fail, and I read on stackoverflow that you should include it so

**Comment by stevenyomi** [comment]
> if you don't include it the PR build check would fail, and I read on stackoverflow that you should include it so

Never heard of that. None of your build checks failed because of that. If it's true, that file would have been added to the repo in the beginning. Just remove it.

File: app/src/main/java/eu/kanade/domain/category/model/Category.kt [hunk 2]
```diff
@@ -18,5 +18,5 @@ data class Category(
...
-    return any { name.equals(it.name, ignoreCase = true) }
+    return any { name.equals(it.name, ignoreCase = false) }
```
Comment (AntsyLich) [inline app/src/main/java/eu/kanade/domain/category/model/Category.kt:21]:
```suggestion
    return any { name == it.name }
```

**Comment by curche** [comment]
> > > including gradle.properites.swp
> > 
> > 
> > Why?
> 
> if you don't include it the PR build check would fail, and I read on stackoverflow that you should include it so

that should be gradle.properties
gradle.properties.swp sounds like a swap/temporary file (of gradle.properties) which usually something like Vim creates when you have a file open
its kinda rare for it to show up and probably why it isn't there in `.gitignore` or just have Vim [create swp files in separate directory](https://vi.stackexchange.com/a/16047)

**Comment by d-najd** [comment]
> > > > including gradle.properites.swp
> > > 
> > > 
> > > Why?
> > 
> > 
> > if you don't include it the PR build check would fail, and I read on stackoverflow that you should include it so
> 
> that should be gradle.properties gradle.properties.swp sounds like a swap/temporary file (of gradle.properties) which usually something like Vim creates when you have a file open its kinda rare for it to show up and probably why it isn't there in `.gitignore` or just have Vim [create swp files in separate directory](https://vi.stackexchange.com/a/16047)

I guess you are correct, looking again at the stackoverflow answer it is refering to gradle.properties, my bad, but I have one question did https://github.com/tachiyomiorg/tachiyomi/actions/runs/3304018563/jobs/5452632515, get canceled because I got a review requesting changes?

**Comment by stevenyomi** [comment]
It was cancelled because you pushed a new commit. Only one PR build check can be run at the same time.

**Pull request merged**